### DISCUSSION TITLE
FuguHub Still an Option?

### DISCUSSION THREAD

**Discussion by holocronology** [MichaIng/DietPi#6077] [Q&A]
FuguHub still appears in the software documentation, but it doesn't appear to be an option for installation via dietpi-software. Was it removed?

**Comment by Joulinar** [comment]
It is still available but might be disabled on your particular device. Can you share following?

```
dietpi-software list | grep Fugu
```

**Comment by MichaIng** [comment]
It is disabled for ARMv8. I'll check if the ARMv6 binary can be used on ARMv8 as well, if it does not depend on shared libraries.

**Reply by MichaIng** [reply]
It is a static binary. @holocronology can you try to remove [this](https://github.com/MichaIng/DietPi/blob/2cc76a5/dietpi/dietpi-software#L796) line in your `/boot/dietpi/dietpi-software` and install FuguHub?

**Reply by Joulinar** [reply]
@MichaIng 

Service is not going to startup on 64bit

```
root@DietPiR5S:~# journalctl -u bdd
-- Journal begins at Tue 2023-01-17 15:34:21 CET, ends at Thu 2023-01-19 14:23:17 CET. --
Jan 19 14:21:04 DietPiR5S systemd[1]: Started FuguHub (DietPi).
Jan 19 14:21:04 DietPiR5S systemd[2191]: bdd.service: Failed to execute /home/bd/FuguHub: No such file or directory
Jan 19 14:21:04 DietPiR5S systemd[2191]: bdd.service: Failed at step EXEC spawning /home/bd/FuguHub: No such file or directory
Jan 19 14:21:04 DietPiR5S systemd[1]: bdd.service: Main process exited, code=exited, status=203/EXEC
Jan 19 14:21:04 DietPiR5S systemd[1]: bdd.service: Failed with result 'exit-code'.
root@DietPiR5S:~#
```

**Reply by MichaIng** [reply]
Ah, I was wrong, `ldd` needs to be executed on an ARMv6/7 system to see that it depends on shared libraries:
```console
root@micha:/tmp# ldd bd/FuguHub
        linux-vdso.so.1 (0x7eb0b000)
        /usr/lib/arm-linux-gnueabihf/libarmmem-${PLATFORM}.so => /usr/lib/arm-linux-gnueabihf/libarmmem-v7l.so (0x76f40000)
        librt.so.1 => /lib/arm-linux-gnueabihf/librt.so.1 (0x76f10000)
        libdl.so.2 => /lib/arm-linux-gnueabihf/libdl.so.2 (0x76ee0000)
        libstdc++.so.6 => /lib/arm-linux-gnueabihf/libstdc++.so.6 (0x76d60000)
        libm.so.6 => /lib/arm-linux-gnueabihf/libm.so.6 (0x76d00000)
        libgcc_s.so.1 => /lib/arm-linux-gnueabihf/libgcc_s.so.1 (0x76cc0000)
        libpthread.so.0 => /lib/arm-linux-gnueabihf/libpthread.so.0 (0x76c90000)
        libc.so.6 => /lib/arm-linux-gnueabihf/libc.so.6 (0x76b60000)
        /lib/ld-linux-armhf.so.3 (0x76f57000)
```
See references to `arm-linux-gnueabihf`, so this cannot work on ARMv8.

**Reply by holocronology** [reply]
Okay, thanks. I saw that it installed as an option on my Pi Zero W but was not showing up on the latest version on my Pi 3B+. 

That's too bad, as it was a nice simple blog/website platform. Need something simpler than WP.

**Reply by Joulinar** [reply]
Theoretically you could run 32bit (ARMv7) DietPi version on your RPi3.

**Reply by MichaIng** [reply]
The one below "Raspberry Pi 2" on the download page.
